packages/html/src/index.test.ts

```diff
@@ -11,6 +11,14 @@ describe('html', () => {
     assert.equal(html`<p>${'cats'} and ${'dogs'}</p>`.toString(), '<p>cats and dogs</p>');
   });
 
+  it('interpolates a number', () => {
+    assert.equal(html`<p>${123}</p>`.toString(), '<p>123</p>');
+  });
+
+  it('interpolates a bigint', () => {
+    assert.equal(html`<p>${123n}</p>`.toString(), '<p>123</p>');
```

Comment:
May be safer to use a value not supported by regular Number, like a 16+ digit number? Just to be safe?

Reply:
Please ignore. I didn't notice it had been merged already, and this is not that important.